thorin-config.cmake

```diff
@@ -31,6 +31,7 @@ FIND_PATH ( THORIN_LIBS_DIR
     NAMES
         ${THORIN_OUTPUT_LIBS}
     PATHS
+        ${THORIN_LIBS_DIR}
```

Comment:
But this is not the case, there is no documentation on THORIN_LIBS_DIR to be an official, externally visible variable. The variable is actually marked as internal variable later on in the file. Otherwise we could add a list of random-generated variables here.